### ISSUE TITLE
Concerning the code efficiency

### ISSUE THREAD

**Issue by anyezhiya368** [alipay/Parameter_Inference_Efficient_PIE#1]
Hi, I am reproducing the result of the FTAI entity typing model and I am very interested in the expected time to achieve the reported result, for the training process seems to take way too long time and cannot reach the reported performance at my end. Many thanks!

**Comment by chocolate9624** [comment]
Yes, the training process will take long time.  The WikiKG90Mv2 is a huge dataset and our implemention is a single GPU version. Thus, a mutl-gpu implemention is needed to accelerate the network tunning  and experiment process.
